docs/api-reference/advanced/boost.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Boost
+---
+
+## Basic Usage
```

Comment:
I've had a few user calls be confused as to whether this functionality was available per-column/per-field. Is this something we should make more explicit here?

Reply:
What is the difference between a column and a field?